src/components/NavigationMenu/NavigationMenu.tsx

```diff
@@ -0,0 +1,430 @@
+import React, { ComponentProps, ElementRef, FC, forwardRef } from 'react'
+import { styled, keyframes, css } from '../../stitches.config'
+import {
+  Root,
+  Trigger,
+  Content,
+  Indicator,
+  Viewport,
+  Item,
+  List,
+  Link,
+} from '@radix-ui/react-navigation-menu'
+import { mainVariants } from '../Button/Button'
+import { ChevronDown } from '../Icons'
+import { paperStyles } from '../Paper'
+
+const enterFromRight = keyframes({
+  from: { transform: 'translateX(200px)', opacity: 0 },
+  to: { transform: 'translateX(0)', opacity: 1 },
+})
+
+const enterFromLeft = keyframes({
+  from: { transform: 'translateX(-200px)', opacity: 0 },
+  to: { transform: 'translateX(0)', opacity: 1 },
+})
+
+const exitToRight = keyframes({
+  from: { transform: 'translateX(0)', opacity: 1 },
+  to: { transform: 'translateX(200px)', opacity: 0 },
+})
+
+const exitToLeft = keyframes({
+  from: { transform: 'translateX(0)', opacity: 1 },
+  to: { transform: 'translateX(-200px)', opacity: 0 },
+})
+
+const scaleIn = keyframes({
+  from: { transform: 'rotateX(-30deg) scale(0.9)', opacity: 0 },
+  to: { transform: 'rotateX(0deg) scale(1)', opacity: 1 },
+})
+
+const scaleOut = keyframes({
+  from: { transform: 'rotateX(0deg) scale(1)', opacity: 1 },
+  to: { transform: 'rotateX(-10deg) scale(0.95)', opacity: 0 },
+})
+
+const fadeIn = keyframes({
+  from: { opacity: 0 },
+  to: { opacity: 1 },
+})
+
+const fadeOut = keyframes({
+  from: { opacity: 1 },
+  to: { opacity: 0 },
+})
+
+const StyledMenu = styled(Root, {
+  position: 'relative',
+  display: 'flex',
+  justifyContent: 'center',
+  width: '100%',
+  zIndex: 1,
+})
+
+const StyledList = styled(List, {
+  all: 'unset',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'stretch',
+  padding: 4,
+  listStyle: 'none',
+  fontSize: '$0',
+  borderRadius: '$3',
+  backgroundColor: '$transparent',
+  border: 'none',
+})
+
+const itemStyles = {
+  padding: '$2 $3',
+  outline: 'none',
+  userSelect: 'none',
+  borderRadius: 4,
+  '&:focus': { position: 'relative', boxShadow: `0 0 0 $0 $colors$primary` },
```

Comment:
use a shadow token - or if this is much better add it to the shadow tokens - don't forget to check how it looks in light and dark.